pages/price-feeds/troubleshoot/svm.mdx

```diff
@@ -0,0 +1,18 @@
+# Troubleshoot Solana Price Feeds Contract
+
+This reference page is designed to help you troubleshoot common issues you may encounter when using Pyth Price Feeds on SVM chains.
+Follow the steps provided below to diagnose and resolve the issue.
+
+### compilation error[E0277]: the trait bound PriceUpdateV2: anchor_lang::AccountDeserialize is not satisfied
+
+This error is caused by an `anchor-lang` version mismatch.
+Make sure the transitive version of `anchor-lang` brough by [pyth-solana-receiver-sdk](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/solana/pyth_solana_receiver_sdk)
+matchers the version of `anchor-lang` of your programs `Cargo.toml`.
+
+You can fix it by following steps:
+
+- Check the version of `anchor-lang` in your `Cargo.toml` (in the example `0.29.0{:js}`) call it `x.y.z`
```

Comment:
why did you remove the links to https://github.com/guibescos/anchor-pyth/blob/broken-build/programs/anchor-pyth/Cargo.toml